tbdex/rfq.go

```diff
@@ -0,0 +1,134 @@
+package tbdex
+
+import (
+	"crypto/sha256"
+	"encoding/json"
+	"fmt"
+
+	"github.com/gowebpki/jcs"
+	"github.com/tbd54566975/web5-go/dids/did"
+)
+
+// RFQKind identifies this message kind
+const RFQKind = "rfq"
+
+// RFQ represents a request for quote message within the exchange.
+type RFQ struct {
+	MessageMetadata MessageMetadata `json:"metadata"`
+	Data            RFQData         `json:"data"`
+	PrivateData     RFQPrivateData  `json:"privateData"`
```

Comment:
pretty sure you'll need to make this `*RFQPrivateData`. Marshal an rfq that has nothing that would be made private and print it out.  `privateData` will be set to its zero value